src/receive_imf.rs

```diff
@@ -291,9 +291,7 @@ pub(crate) async fn receive_imf_inner(
     if let Some(ref sync_items) = mime_parser.sync_items {
         if from_id == ContactId::SELF {
             if mime_parser.was_encrypted() {
-                if let Err(err) = context.execute_sync_items(sync_items).await {
-                    warn!(context, "receive_imf cannot execute sync items: {err:#}.");
-                }
+                context.execute_sync_items(sync_items).await;
```

Comment:
The problem is that errors that are critical at the lower level (not sure that "critical" is the correct wording, but they are at least bubbled up) aren't critical at this level, e.g. some logic errors or ones caused by an invalid input. So, if we call some function that can fail for different reasons, incl. database write failures, we should always ignore errors from it, but the comment on `execute_sync_items()` said that db failures shouldn't be ignored, afaiu, because executing next items has no sense (they also would fail). This problem can't be solved w/o typed errors anyway, so i suggest not to complicate logic for now and just ignore errors in the item execution loop.